vendor/github.com/mattn/go-shellwords/.travis.yml

```diff
@@ -2,7 +2,6 @@ language: go
 go:
   - tip
 before_install:
-  - go get github.com/axw/gocov/gocov
```

Comment:
It does seem like there is some bug in trash. Perhaps you're using an older version.

Using a tag that doesn't change is actually intended. The results of running trash should be deterministic and not depend on the master branch of a repo. To update go-shellwords in the future we'd have to update `trash.conf` to point to a new tag.